packages/vscode-extension/lib/preview.js

```diff
@@ -14,6 +14,21 @@ function stringifyProps(obj) {
   return keyValuePairs.join(" ");
 }
 
+function getComponentName({ type }) {
+  const name = type.name;
+  const isClassOrFunctionComponent = name !== undefined;
+  if (isClassOrFunctionComponent) {
+    return name;
+  }
+
+  const isForwardedRef = type["$$typeof"] === Symbol.for("react.forward_ref");
+  if (isForwardedRef) {
+    return "(forwarded ref)";
```

Comment:
I think it would be more usefull for the user to see a generic name like "Component ${index}" here, because it could help them navigate the menu between multiple components.